idarling/core/events.py

```diff
@@ -336,16 +336,12 @@ def __call__(self):
                 ti = get_idati()
                 ordinal = ctypes.c_uint32(py_ord)
                 ntf_flags = ctypes.c_int(ida_typeinf.NTF_REPLACE)
-                name = Event.encode_bytes(name)
-                type = Event.encode_bytes(type)
-                fields = Event.encode_bytes(fields)
-                cmt = Event.encode_bytes(cmt)
                 fieldcmts = Event.encode_bytes(fieldcmts)
```

Comment:
`fieldcmts` is encoded twice!

Reply:
FYI: No need to force push on PRs, all commits as squashed into one during the merge.